docs/developer/HootenannySecurity.asciidoc

```diff
@@ -5,7 +5,7 @@
 
 Hoot services are now authenticated with OAuth2 which adds a config file oauth2-client.properties that will potentially need to be tweaked per environment.
 
-This file is found in the hoot-services.war file under the path `WEB-INF/classes/oauth2-client.properties`.
+This file is found in the hoot-services.war file under the path `/var/lib/tomcat8/webapps/hoot-services/WEB-INF/classes/oauth2-client.properties`.
```

Comment:
@brianhatchl is there a better way to do this?  I don't think a full path (that could change) is the right way to go.
```suggestion
This file is found in the hoot-services.war file under the path `<tomcat home>/webapps/hoot-services/WEB-INF/classes/oauth2-client.properties`.
```
or maybe
```suggestion
This file is found in the hoot-services.war file under the path `<webapps home>/hoot-services/WEB-INF/classes/oauth2-client.properties`.
```